tools/ci_build/github/android/build_aar_package.py

```diff
@@ -99,6 +100,12 @@ def _build_aar(args):
         abi_build_dir = os.path.join(intermediates_dir, abi)
         abi_build_command = [*base_build_command, "--android_abi=" + abi, "--build_dir=" + abi_build_dir]
 
+        if qnn_android_build:
+            qnn_home = os.getenv("QNN_HOME")
+            qnn_sdk_version = os.getenv("QNN_SDK_VERSION")
```

Comment:
nit: can we read this from QNN_HOME to ensure the version is correct? 

<QNN_HOME>/sdk.yaml has 'version: 2.26.2' which should be easy to parse for